### PR TITLE
Armis Bug Fix

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/37070

## Description
Fixing incidents duplicates fetch from Armis integration

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
Added unit test to indicate duplicate failure.
Test unit failure before fix
https://app.circleci.com/pipelines/github/demisto/content/90250/workflows/4640e097-6412-4177-8723-df22c400b8f3/jobs/373850

**Comment by tomneeman151293** [comment]
This is PR for a possible fix. I would like to send it to the customer to try (I'll change the ID of the YML when i send it)
@dantavori @ShahafBenYakir waiting for your approval to send the unified YML to the customer.

**Review by dantavori — APPROVED** [review]
merge once customer approves the fix

**Pull request merged**
